src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -114,9 +114,14 @@ COOP_PINVOKE_HELPER(HANDLE, RhGetModuleFromPointer, (PTR_VOID pPointerVal))
 
 COOP_PINVOKE_HELPER(HANDLE, RhGetModuleFromEEType, (EEType * pEEType))
 {
-#if CORERT
+#if CORERT  
     return (HANDLE)(pEEType->GetTypeManager());
 #else
+#if EETYPE_MODULE_MANAGER
```

Comment:
In CORERT, we always have a non-null TypeManager for EEType. In ProjectN, we have a non-null TypeManager for ProjectX EEType but a null one for ProjectN EEtype.